base/src/main/java/com/smartdevicelink/proxy/rpc/AppCapability.java

```diff
@@ -0,0 +1,50 @@
+package com.smartdevicelink.proxy.rpc;
+
+
```

Comment:
Based on the mobile_API.xml, this file should have the `/** @since 7.0 */` annotation.

Reply:
> Based on the mobile_API.xml, this file should have the `/** @since 7.0 */` annotation.

@kostyaBoss I dont see the annotation `/** @since 7.0 */` at the top. Can you please add it whenever you get the chance?